js/Features.js

```diff
@@ -46,6 +46,13 @@ var Features = function (config) {
         );
     }
 
+    if (semver.gte(config.apiVersion, "1.21.0")) {
+        features.push(
+            {bit: 25, group: 'rxMode', mode: 'group', name: 'RX_SPI', haveTip: true},
```

Comment:
I am not quite happy with how features are 'abused' at the moment for groups of modes that are mutually exclusive, like RX selection. I think we should try to come up with something better, and move these things out of the features set.